spec/index.html

```diff
@@ -516,7 +516,7 @@ <h3>Blank nodes</h3>
 	<li> [I+A](x)=I(x) when x is a <a>name</a>. </li>
 	<li> [I+A](x)=A(x) when x is a blank node. </li>
 	<li> [I+A](x)=RE( [I+A](x.s), [I+A](x.p), [I+A](x.o) ) when x is a triple term, where x.s, x.p, and x.o are the first, second, and third components of x, respectively. </li>
-	<li> [I+A](x)=true when x is a triple, x.s, x.p, and x.o are the first, second, and third components of x, respectively, and [I+A](x.p) is in IP and the pair &lt;[I+A](x.s),[I+A](x.o)&rt; is in IEXT([I+A](x.p)). </li>
+	<li> [I+A](x)=true when x is a triple, x.s, x.p, and x.o are the first, second, and third components of x, respectively, and [I+A](x.p) is in IP and the pair &lt; [I+A](x.s),[I+A](x.o) &gt; is in IEXT([I+A](x.p)). </li>
```

Comment:
```suggestion
	<li> [I+A](x)=true when x is a triple, x.s, x.p, and x.o are the first, second, and third components of x, respectively, and [I+A](x.p) is in IP and the pair &lt; [I+A](x.s), [I+A](x.o) &gt; is in IEXT([I+A](x.p)). </li>
```
Consistent use of spaces with line above.